This PR and release of 3.0 is on hold until Monday when I can resolve some newly introduced bugs caught after updating PyNWB to account for breaking changes in HDMF and then running PyNWB tests, e.g., 

```
======================================================================
FAIL: test_catch_dup_name (test_extension.TestCatchDupNS)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Ryan\Documents\NWB\pynwb\tests\unit\test_extension.py", line 172, in test_catch_dup_name
    type_map.load_namespaces(os.path.join(self.tempdir, self.ns_path2))
AssertionError: UserWarning not triggered

======================================================================
FAIL: test_simple (test_file.TestCacheSpec)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Ryan\Documents\NWB\pynwb\tests\unit\test_file.py", line 471, in test_simple
    nwbfile = reader.read()
AssertionError: UserWarning not triggered
```
and many of:
```
  File "c:\users\ryan\documents\nwb\hdmf\src\hdmf\build\objectmapper.py", line 965, in __add_datasets
    self.__check_quantity(attr_value, spec, container)
  File "c:\users\ryan\documents\nwb\hdmf\src\hdmf\build\objectmapper.py", line 661, in __check_quantity
    n = len(attr_value)
  File "c:\users\ryan\documents\nwb\hdmf\src\hdmf\data_utils.py", line 668, in __len__
    return len(self.data)
TypeError: object of type 'DataChunkIterator' has no len()
```
and related issues from that line.

--- a/.github/workflows/workflow.yml
+++ b/.github/workflows/workflow.yml
@@ -1,9 +1,12 @@
 name: Run coverage
 on:
   push:
-    tags-ignore:
-    - latest
-    - latest-tmp
+    branches:
+    - *
+    tags:
+    - *
+    - '!latest'
+    - '!latest-tmp'
 jobs:
   run:
     runs-on: ${{ matrix.os }}

--- a/.github/workflows/workflow.yml
+++ b/.github/workflows/workflow.yml
@@ -1,5 +1,9 @@
 name: Run coverage
-on: [push]
+on:
+  push:
+    tags-ignore:
+    - latest
+    - latest-tmp
 jobs:
   run:
     runs-on: ${{ matrix.os }}

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,6 @@
 # HDMF Changelog
 
-## HDMF 3.0.0 (December 7, 2020)
+## HDMF 3.0.0 (December 8, 2020)
 
 ### Breaking changes
 - Drop support for Python 3.5. @ajtritt (#459)

